README.md

```diff
@@ -59,13 +59,21 @@ Must be a list of *tensor specification keys*.
 
   *tensor specification keys*:
   - `name` tensor name
-  - `axes` string of axes identifiers, e.g. btczyx
   - `data_type` data type (e.g. float32)
   - `data_range` tuple of (minimum, maximum)
+  - `axes` string of axes identifying characters from: btczyx
```

Comment:
Do we have a place to give detailed definition for the axes and the meaning? Are we allowed to give a custom letter to it?

Reply:
I think we should be as restrictive with these letters as possible to keep them meaningful/useful from a consumer software perspective, e.g restrict them to btczyx. We can have a (separate) discussion on the axes keys (use HW, instead of xy, etc...). I'd prefer to delay that for 0.3.1+, for now in a given input the `description` field could add specific meaning in the model context for humans? I feel an `axes_description` field would go a bit too far anyway, but again, let's leave that for future discussion if necessary.

Reply:
I do agree that for now, it is too ambitious to get this kind of specifications. I think it was already discussed at some point but outputs might also be better described with rows and columns. While in NumPy arrays those could be understood as HW, displaying them as tables could need a different kind of description.